pkgs/servers/sql/postgresql/pg_config.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
This can go I'd say.

`writeShellApplication` already adds a shebang on its own and this one doesn't exist on NixOS, so there's no point in keeping it, even for testing purposes.